sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Release History
 
-## 1.1.7 (Unreleased)
+## 2.0.0-beta.1 (Unreleased)
 
 - Internal improvement - Previously, each `RequestResponseLink.sendRequest` call adds an "onMessage" listener to the `ReceiverEvents.message` event and keeps discarding the responses that did not match the request-id and returns the response if matched. Adding many listeners would also result in a warning such as `MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 message listeners added to [Receiver]. Use emittr.setMaxListeners() to increase limit`.
```

Comment:
There is no reason why v1 users of `core-amqp` should not reap the benefits of this improvement. Can we consider releasing a 1.1.7 of core-amqp tomorrow and then move on to v2?

Reply:
If we are going down this route, then we should do have a PR that updates the release date in the changelog merged before merging this PR

Reply:
Actually, if we are going to do a 1.1.7 with just the internal improvement, then that would be a hotfix as #11973 has already made changes to it which includes the error codes which is still under discussion. So, this PR need not be blocked by the 1.1.7 decision

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/12125